extensions/amp-analytics/0.1/vendors.js

```diff
@@ -41,6 +41,11 @@ export const ANALYTICS_CONFIG = {
       'scrollHeight': 'SCROLL_HEIGHT',
       'screenWidth': 'SCREEN_WIDTH',
       'screenHeight': 'SCREEN_HEIGHT'
+      // TODO: Requested by Parse.ly
+      //   screenAvailWidth (screen.availWidth)
```

Comment:
I think you can remove the first three items here. @rudygalfi has opened #1503.

cc @cramforce for the last item. I am not sure if/how we'll do the session stuff. I think it requires some kind of storage on the client side.